src/integration/UsesTmpDir.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Flarum\Testing\integration;
+
+trait UsesTmpDir
+{
+    protected $DEFAULT_TMP_DIR = __DIR__.'/tmp';
+
+    public function tmpDir() {
+        return realpath(getenv('FLARUM_TEST_TMP_DIR_LOCAL') ?: getenv('FLARUM_TEST_TMP_DIR') ?: static::$DEFAULT_TMP_DIR);
+    }
+
+    public function setupTmpDir() {
```

Comment:
It was, but that uses this function.

Reply:
I'm probably missing something. It didn't use it before and it still created the tmp dir, so why does it need to use this method now ?

Reply:
Previously, the source code included a skeleton structue via `.gitkeep` files (removed in https://github.com/flarum/testing/pull/2/commits/d1ee17c0666cf0c39d03afe8a46cadbd891c3404 as part of this PR). Instead of that, this approach will automatically create the tmp dir structure in whatever folder is selected.

Reply:
Makes sense, thank you for the clarification, I totally missed that.